### ISSUE TITLE
Application startup fails when init.lua contains "local howl = require 'howl'"

### ISSUE THREAD

**Issue by bugabinga** [howl-editor/howl#455]
When the line 

```lua
local howl = require 'howl'
```

is present in `~/.howl/init.lua`, the application fails to start (no window) and prints following error to _stdout_:

```sh
../lib/howl/init.lua:41: bad argument #1 to 'ipairs' (table expected, got nil)
../lib/howl/log.moon:32: module 'howl.app' not found:
	no field package.preload['howl.app']
	no file 'howl/lib/howl/app.lua'
	no file 'howl/lib/howl/app/init.lua'
	no file 'howl/lib/ext/howl/app.lua'
	no file 'howl/lib/ext/howl/app/init.lua'
	no file 'howl/lib/ext/moonscript/howl/app.lua'
	no file 'howl/lib/ext/moonscript/howl/app/init.lua'
	no file '/usr/local/share/howl/lib/howl/app.lua'
	no file '/usr/local/share/howl/lib/howl/app/init.lua'
	no file '/usr/local/share/howl/lib/ext/howl/app.lua'
	no file '/usr/local/share/howl/lib/ext/howl/app/init.lua'
	no file '/usr/local/share/howl/lib/ext/moonscript/howl/app.lua'
	no file '/usr/local/share/howl/lib/ext/moonscript/howl/app/init.lua'
	no file './howl/app.lua'
	no file '/usr/local/share/luajit-2.1.0-beta3/howl/app.lua'
	no file '/usr/local/share/lua/5.1/howl/app.lua'
	no file '/usr/local/share/lua/5.1/howl/app/init.lua'
	no file './howl/app.so'
	no file '/usr/local/lib/lua/5.1/howl/app.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'
	no file './howl.so'
	no file '/usr/local/lib/lua/5.1/howl.so'
	no file '/usr/local/lib/lua/5.1/loadall.so'

```

It seems _arg\_vector_ in https://github.com/howl-editor/howl/blob/98a662547cac0eadea7a2897e903a05462953a13/lib/howl/init.lua#L41 is nil.
Which means _argv_ in https://github.com/howl-editor/howl/blob/98a662547cac0eadea7a2897e903a05462953a13/lib/howl/init.lua#L178 is nil.

I realise, that nowhere does the [documentation] actually advises to do this.
But I am used from other applications configured via lua, that globals can be rebound with `require`, so I did it out of habit.

IMO, this is __not critical__ and should you flag this as _WONT-FIX_ I would like to add a paragraph to the docs explaining the reasons.
On the other hand, it is troublesome, that configs can break the application.
What are your goals in this regard?

---
OpenBSD-current
Howl master branch

[documentation]: https://howl.io/doc/manual/configuration.html

**Comment by kirbyfan64** [comment]
I think it's sort of a known issue that your init file can stop Howl from starting...

**Comment by shalabhc** [comment]
Looks like this was fixed in 77b38aba5
